src/EntityFramework/Core/Common/CommandTrees/ExpressionBuilder/DbExpressionBuilder.cs

```diff
@@ -208,33 +208,39 @@ public static DbGroupExpressionBinding GroupBindAs(this DbExpression input, stri
         /// <param name="argument">The argument over which the aggregate function should be calculated.</param>
         /// <exception cref="T:System.ArgumentNullException">function or argument null.</exception>
         /// <exception cref="T:System.ArgumentException">function is not an aggregate function or has more than one argument, or the result type of argument is not equal or promotable to the parameter type of function.</exception>
-        public static DbFunctionAggregate Aggregate(this EdmFunction function, DbExpression argument)
+        public static DbFunctionAggregate Aggregate(this EdmFunction function, IEnumerable<DbExpression> arguments)
```

Comment:
please add new overload rather than modifying existing one to avoid breaking change in public API